From 0d52defd38fc0f13ea518ced40490412dc4ce429 Mon Sep 17 00:00:00 2001
From: Gaku Inami <gaku.inami.xw@bp.renesas.com>
Date: Thu, 17 Mar 2016 16:24:32 +0900
Subject: [PATCH 1462/2066] arm64: dts: r8a7795-es1: Add the emergency node in
 thermal-zones

This patch adds emergency shutdown support for R8A7795 SoC.

[gaku.inami.xw: disable emergency node to avoid thermal zone warning]
Signed-off-by: Gaku Inami <gaku.inami.xw@bp.renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
[zou:Original patch taken from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git
 v4.9/rcar-3.5.1]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi |   10 ++++++++++
 1 files changed, 10 insertions(+), 0 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
index 9e26dd1..96b38aa 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
@@ -2496,6 +2496,16 @@
 		};
 
 		thermal-zones {
+			emergency {
+				polling-delay = <1000>;
+				on-temperature = <110000>;
+				off-temperature = <95000>;
+				target_cpus = <&a57_1>,
+					      <&a57_2>,
+					      <&a57_3>;
+				status = "disabled";
+			};
+
 			sensor_thermal1: sensor-thermal1 {
 				polling-delay-passive = <250>;
 				polling-delay = <0>;
-- 
1.7.5.4

